charts/seed-machines/charts/packet-machineclass/templates/packet-machineclass.yaml

```diff
@@ -0,0 +1,39 @@
+{{- range $index, $machineClass := .Values.machineClasses }}
```

Comment:
Delete this file (the whole `packet-machineclass` (respectively, the whole `seed-machines) directory).